website/pages/en/sunrise.mdx

```diff
@@ -0,0 +1,117 @@
+
+With The Graph Network, your subgraph is public and anyone can query it openly, which increases the usage and network effects of your dapp. With a centralized indexing solution, the subgraph is private to the centralized provider.
+
+Here's a detailed breakdown of the benefits of The Graph over centralized hosting:
+
+Resilience and Redundancy: Decentralized systems are inherently more robust and resilient due to their distributed nature. Data isn't stored on a single server or location. Instead, it's served by hundreds of independent Indexers around the globe. This reduces the risk of data loss or service interruptions if one node fails, leading to exceptional uptime (99.99%).
+
+Quality of Service: In addition to the impressive uptime, The Graph Network features a ~106ms median query speed (latency), and higher query success rates compared to hosted alternatives. Read more in [this blog](https://thegraph.com/blog/qos-the-graph-network/).
+
+Censorship Resistance: Centralized systems can become targets for censorship, either through regulatory pressures or network attacks. In contrast, decentralized systems, due to their dispersed architecture, are much harder to censor, ensuring continuous data availability.
+
+Transparency and Trust: Decentralized systems operate openly, enabling anyone to independently verify the data. This transparency builds trust among network participants, as they can verify the system's integrity without relying on a central authority.
+
+Just as you've chosen your blockchain network for its decentralized nature, security, and transparency, opting for The Graph Network is an extension of those same principles. By aligning your data infrastructure with these values, you ensure a cohesive, resilient, and trust-driven development environment.
+
+### Will my hosted service subgraph be supported by the upgrade Indexer?
+
+Yes, the upgrade Indexer will support all subgraphs published during the upgrade window.
+
+However, some subgraphs may not be eligible for indexing rewards, and as a result, may have difficulty attracting further Indexers. For example, indexing rewards may not be available for subgraphs on certain chains. Members from these blockchain communities are encouraged to integrate their chain through the [Chain Integration Process](/chain-integration-overview/).
+
+## WHat is an upgrade Indexer?
+
+### What is an upgrade Indexer? What does "upgrade Indexer" mean?
+
+It is designed to improve the experience of upgrading subgraphs from the hosted service to The Graph Network and supporting new versions of existing subgraphs that have not yet been indexed.
+
+The upgrade Indexer is aimed at bootstrapping chains that do not yet have indexing rewards on the network, as well as a fallback for new subgraph versions. The goal is to ensure that an Indexer is available to serve queries as quickly as possible after a subgraph is published.
```

Comment:
The title says "_an_ upgrade Indexer" while this says "_The_ upgrade Indexer", so I find it a bit confusing... is there just one, or are there multiple upgrade Indexers?

Reply:
So, there is only one initially but other Indexers are encouraged to become upgrade Indexers as well. We already have 2 or 3 folks interested, with more potentially joining. However, there is no hard commitment. Hence, the hesitation around this.

This language comes directly from a google doc that _many_ people have reviewed, so I lean toward keeping the phrasing as is, and changing later if needed. Does that make sense?